docker/apply/forms-shelf/find-translators-interpreters.json

```diff
@@ -1,3416 +1,3416 @@
 {
```

Comment:
Does updating this file related to the case sensitivity issue or is this for something else?

Reply:
nope, this is more for the unmoderated UR testing that was being carried out